WebDavServer.WebApi/Controllers/WebDavController.cs

```diff
@@ -48,8 +48,17 @@ public async Task GetAsync(string? path, CancellationToken cancellationToken = d
 
         [ApiExplorerSettings(IgnoreApi = true)]
         [AcceptVerbs("PROPFIND")]
-        public async Task<string> PropfindAsync(string? path, CancellationToken cancellationToken)
+        public async Task<IActionResult> PropfindAsync(string? path, CancellationToken cancellationToken)
         {
+            if (path is not null && (path.Contains("desktop.ini") ||
```

Comment:
Why do these files need to be hidden?

Reply:
Annoying windows explorer-only stuff that gets checked on a file system that would never contain them *shrug :)

Reply:
Place the list of files in a separate class, something like this

```
public static class WebDavConsts {

public static string[] ExcludeHiddenFiles = new[] { ... }
}

...
// use
if (WebDavConsts.ExcludeHiddenFiles.Any(path.Contains)) { ... }

```